packages/synapse-react-client/src/components/QueryContext/QueryContext.tsx

```diff
@@ -1,30 +1,14 @@
 import React, { createContext, useContext } from 'react'
-import { SynapseClientError } from '../../utils'
-import {
-  AsynchronousJobStatus,
-  ColumnModel,
-  QueryBundleRequest,
-  QueryResultBundle,
-} from '@sage-bionetworks/synapse-types'
+import { QueryBundleRequest } from '@sage-bionetworks/synapse-types'
 import { ImmutableTableQueryResult } from '../../utils/hooks/useImmutableTableQuery/useImmutableTableQuery'
 import { ReadonlyDeep } from 'type-fest'
+import { TableQueryUseQueryOptions } from '../QueryWrapper/TableQueryUseQueryOptions'
+import { LockedColumn } from '../../utils'
 
-type OptionalQueryBundleRequestFields = keyof Pick<
-  QueryResultBundle,
-  | 'queryResult'
-  | 'queryCount'
-  | 'selectColumns'
-  | 'maxRowsPerPage'
-  | 'columnModels'
-  | 'facets'
-  | 'sumFileSizes'
-  | 'combinedSql'
->
-
-export type QueryContextType<
-  TIncludedFields extends OptionalQueryBundleRequestFields = never,
-  ExcludeOtherFields extends true | false = false,
-> = {
+export type QueryContextType = {
+  isInfinite: boolean
+  entityId: string
+  versionNumber: number | undefined
```

Comment:
Moved `entityId`/`versionNumber` to context so components can fetch the entity if needed (entity data was previously in Jotai)